repos/spack_repo/builtin/packages/py_superqt/package.py

```diff
@@ -15,15 +15,24 @@ class PySuperqt(PythonPackage):
 
     license("BSD-3-Clause", checked_by="A-N-Other")
 
+    version("0.7.6", sha256="822fdba71dc391929c9d3db839f78ca2a861e2f2876926f969a288dfb2a9787e")
     version("0.6.1", sha256="f1a9e0499c4bbcef34b6f895eb57cd41301b3799242cd030029238124184dade")
 
+    variant("iconify", default=False, description="Use Iconify icons keys")
+
     depends_on("python@3.8:", type=("build", "run"))
     depends_on("py-hatchling", type="build")
     depends_on("py-hatch-vcs", type="build")
 
     depends_on("py-packaging", type=("build", "run"))
     depends_on("py-pygments@2.4:", type=("build", "run"))
+    depends_on("py-qtpy@2.4:", when="@0.7.6:", type=("build", "run"))
     depends_on("py-qtpy@1.1:", type=("build", "run"))
-    depends_on("py-typing-extensions@3.7.4.3:", type=("build", "run"))
+    depends_on("py-typing-extensions@4.12:", when="@0.7.6: ^python@3.13:", type=("build", "run"))
```

Comment:
Is this actually a `py-superqt` constraint, or is this a misplaced constraint that should be on `py-typing-extensions`? If it's the latter, we can remove it here because adding it there will ensure the compatibility we need.